site/snippets/layouts/footer.php

```diff
@@ -80,11 +80,11 @@
 					<li>
 						<p class="font-bold mb-1">Kirby</p>
 						<ul class="footer-menu-2">
-							<li><a href="/security">Security</a></li>
-							<li><a href="/privacy">Privacy</a></li>
-							<li><a href="/license">License</a></li>
-							<li><a href="/press">Presskit</a></li>
-							<li><a href="/contact">Contact</a></li>
+							<li><a href="<?= page('security')->menuUrl() ?>">Security</a></li>
+							<li><a href="<?= page('privacy')->menuUrl() ?>">Privacy</a></li>
+							<li><a href="<?= page('license')->menuUrl() ?>">License</a></li>
+							<li><a href="<?= page('press')->menuUrl() ?>">Presskit</a></li>
+							<li><a href="<?= page('contact')->menuUrl() ?>">Contact</a></li>
```

Comment:
`Suggestion:` To keep the footer and main menus DRY-er, we could use a snippet which we just pass something like

```php
[
    'For developers' => page('features/developers'),
    'For designers' => page('features/designers'),
    'Feedback' => 'https://feedback.getkirby.com'
]
```

And the snippet renders each `<li>` depending whether it contains a page object or a string (for external) differently).